packages/api/tests/routes.test.js

```diff
@@ -1,9 +1,34 @@
-const request = require('supertest')
-const app = require('../server')
-describe('GET Endpoints', () => {
-  it('should create a new GET request', async () => {
+const request = require('supertest');
+const app = require('../server');
+const db = require('../models');
+
+describe('GET Nomination Endpoint', () => {
+  let nomination;
+
+  beforeAll(() => {
+    nomination = db.Nomination.build({ id: 'b5a27641-d76b-4ab0-9cef-8bf9eb9e8fab' });
+    return nomination.save();
+  });
+
+  afterAll(() => {
+    nomination.destroy();
+  });
+
+  it('when nomination exist it should return a 200', async () => {
```

Comment:
Language nitpick:

`'returns a 200 when nomination exists'`

Very minor thing here, but I like to have nice grammar in the test output, and "it when nomination exists it should return..." reads kinda clunkily to me.